x-pack/plugin/ml/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/ml/transforms/PainlessDomainSplitIT.java

```diff
@@ -190,8 +189,7 @@ public void testIsolated() throws Exception {
 
         Pattern pattern = Pattern.compile("domain_split\":\\[(.*?),(.*?)\\]");
 
-        Map<String, Object> params = new HashMap<>(DomainSplitFunction.params.size() + 1);
-        params.putAll(DomainSplitFunction.params);
```

Comment:
nit: rename this class PainlessDomainSplitIT to DomainSplitIT

Reply:
I'm happy to rename it, but this function still only applies to painless. Is there another reason you have for the rename?